.github/dependabot.yml

```diff
@@ -26,6 +26,10 @@ updates:
       csv:
         patterns:
         - "csv*"
+      futures:
+        patterns:
+        - "futures"
+        - "futures-util"
```

Comment:
note I didn't use `futures-*` so that we don't make an assumption here and end up adding some crate we don't want grouped with these two.